app/brave_generated_resources.grd

```diff
@@ -490,6 +490,9 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_EXTENSIONS_BRAVE_PRIVATE_AND_TOR_WARNING" desc="Displayed in extensions management UI after an extension is selected to be run in private mode and Tor mode.">
         To disable this extension in private mode and Tor mode, unselect this option.
       </message>
+      <message name="IDS_EXTENSIONS_BRAVE_HOSTED" desc="Displayed in extensions management UI if an extension is hosted on the Brave server.">
+        Brave-hosted
```

Comment:
do we want the `-` here? It feels a bit weird

Reply:
I'd asked for this, yeah, can omit the `-`.

Reply:
Hmm, actually looks like `Brave-hosted` is more correct, since it's 2 words working as a single adjective, really we're saying `Brave-hosted extension`.

Reply:
I don't have strong feelings about this, it just looks slightly off to me. Let's see what the string reviewers reckon :smile:

Reply:
`Hosted by Brave` could be ok too, but don't have strong feelings either.